test/smoke/generated/web-resource-manager/src/models/parameters.ts

```diff
@@ -908,7 +908,7 @@ export const name1: OperationParameter = {
 };
 
 export const typeParam: OperationParameter = {
-  parameterPath: "typeParam",
+  parameterPath: "type",
```

Comment:
@joheredi  I am not sure if this is the right fix. In this case, we changed the parameterPath from typeParam to type.
And type is simple string type, we don't have it defined in the models/index.ts. Can you confirm?

Reply:
Parameter path always needs to match the parameter name passed to the operation in operations or client. In this clase this needs to be `typeParam`

![image](https://user-images.githubusercontent.com/21109913/159978361-76858dab-dc90-4b0d-9551-8bf9a9f8bb4d.png)

Reply:
changed back